queue/src/fiskaltrust.Middleware.Localization.QueueDE/JournalProcessorDE.cs

```diff
@@ -151,7 +152,11 @@ private async IAsyncEnumerable<JournalResponse> ProcessTarExportFromTSEAsync(Jou
         {
             var exportSession = await _deSSCDProvider.Instance.StartExportSessionAsync(new StartExportSessionRequest()).ConfigureAwait(false);
             var sha256CheckSum = "";
-            using (var memoryStream = new MemoryStream())
+
+            byte[] chunk;
```

Comment:
Did you test where this file is created? In the service folder, or in the same directory as the Launcher is located?

Reply:
its in the same folder as the launcher and deleted afterward.